<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBox</class>
 <widget class="QWidget" name="MainBox">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>UPS:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_ups"/>
      </item>
      <item>
       <widget class="QToolButton" name="btn_run">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>input.frequency:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLCDNumber" name="lcd_ups_temperature"/>
     </item>
     <item row="9" column="1">
      <widget class="QLCDNumber" name="lcd_input_voltage_nominal"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>output.voltage:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>battery.voltage.high:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>battery.charge:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>battery.voltage.low:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLCDNumber" name="lcd_battery_voltage_low"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>input.voltage:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLCDNumber" name="lcd_output_voltage"/>
     </item>
     <item row="7" column="1">
      <widget class="QLCDNumber" name="lcd_input_voltage"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>input.voltage.nominal:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcd_battery_charge"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>battery.voltage:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLCDNumber" name="lcd_input_frequency"/>
     </item>
     <item row="1" column="1">
      <widget class="QLCDNumber" name="lcd_battery_voltage"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>input.voltage.fault:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLCDNumber" name="lcd_input_voltage_fault"/>
     </item>
     <item row="5" column="1">
      <widget class="QLCDNumber" name="lcd_battery_voltage_nominal"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>ups.temperature:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>battery.voltage.nominal:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLCDNumber" name="lcd_battery_voltage_high"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
